src/main/resources/db/migration/V1__GCP_tilganger.sql

```diff
@@ -0,0 +1,11 @@
+DO
+$$
+    BEGIN
+        IF EXISTS
+            (SELECT 1 from pg_roles where rolname = 'cloudsqliamuser')
```

Comment:
Typo? "rolename"

Reply:
hmm, nei. https://www.postgresql.org/docs/8.1/view-pg-roles.html

Dette er forøvrig samme som ligger i alle gcp db appene

Reply:
Noen skrev feil for mange år siden, så har de bare måtte fortsette 😁